style/style.css

```diff
@@ -242,9 +351,9 @@
       box-shadow: 0px 14px 40px #00000029;
       padding: 3rem;
       border-radius: 50%;
-      position: absolute;
-      top: 37rem;
-      right: 48rem;
+      margin-top: -15rem;
+      margin-right: -15rem;
+      z-index: 100;
```

Comment:
Pretty cringe here. Because I add a container on the nav to be able to import the menu, the "space-beetwen" is not working anymore. I fixed it by adding some marges. Cringe, but efficient.